instagram/src/components/PostContainer/PostFooter.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+export default class PostFooter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      likes: props.likes
+    }
+
+  }
+
+  addLikes = () => {
+    this.setState({
+      likes: this.state.likes + 1
```

Comment:
Do not directly mutate state. On line 16 it would be better to use `this.setState(prevState` =>{
likes: prevState.likes + 1})`